tools/docker/tests/test_flows.sh

```diff
@@ -71,22 +71,34 @@ send_CAPI_command() {
     capi_command_reply=$(echo -e "$capi_command_result" | awk 'NR==2')
 }
 
+check_log() {
+    # Check for a certain message in a log file
+    # $1: device to check (gateway, repeater1, repeater2)
+    # $2: program to check (controller, agent, agent_wlan0, ...)
+    # rest: arguments to grep. -q is always added.
+    device="$1"; shift
+    program="$1"; shift
+    check grep -q "$@" "${rootdir}/logs/${device}/beerocks_${program}.log"
```

Comment:
So long as no test need to look at the `transport.log` or `local_bus.log`, these don't have the `beerocks_` prefix for some reason

Reply:
Good point. However, the chance that we need to grep those two log files is rather small, and we're anyway converting the test flows to a python-based setup so this will no longer be relevant. So I'm going to leave it as it is now, and we can still fix it later if needed.